test/entry.spec.js

```diff
@@ -135,9 +135,10 @@ apis.forEach((IPFS) => {
 
     describe('toMultihash', () => {
       it('returns an ipfs hash', async () => {
-        const expectedHash = 'QmZHattK3ayuXJudvUcQqxM6Mh2nUzxjBKNgLW22cFYMKg'
+        const expectedHash = 'Qmcga9V6D7EPVQTPWjyjFUP6NoGivhymLGi4f9VMGQA24x'
         const entry = await Entry.create(ipfs, null, 'A', 'hello')
         const hash = await Entry.toMultihash(ipfs, entry)
+        assert.equal(entry.hash, expectedHash)
         assert.equal(hash, expectedHash)
```

Comment:
The assertion here was passing before. If a hash change is needed to make it pass again, then it means something was changed that changes the shape of the entry object. Do you have insight to what changed?

Reply:
Clarifying here re. "shape of the object" (which is not entirely accurate): the entry that gets saved to IPFS has changed either in its shape or its data is different before.

Reply:
> When toMultihash is called by Entry.create, it takes an entry with no .hash and returns a multihash that is then assigned to the entry later

Yes, the difference is that when we call `.toMultihash` the second time, with the created entry, it's got a hash already, that makes the hash returned by `.toMultihash` to be different from `entry.hash`